applications/tasks/src/tasks_maintenance.erl

```diff
@@ -110,13 +108,6 @@ start(TaskId) ->
         {'error', Reason} -> print_error(Reason)
     end.
 
--spec restart(text()) -> 'no_return'.
-restart(TaskId) ->
```

Comment:
"Never meant to" doesn't mean "never will". I am showing you a use case where I start a job via API, it fails because the code didn't work properly, I adjust the code and restart the task. Its really quite simple, doesn't impact existing functionality, only exists for admins via SUP...